tests/core_test/containers_test/repertoire_test.py

```diff
@@ -46,9 +47,12 @@ def test_repertoire() -> None:
     fake_genotypes = jnp.ones(shape=(batch_size, genotype_size))
     fake_fitnesses = jnp.zeros(shape=(batch_size,))
     fake_descriptors = jnp.array([[0.1, 0.1], [0.9, 0.9]])
+    fake_extra_scores: ExtraScores = {}
```

Comment:
given the optional type of the argument in the add function, I think it would make more sense to define as None by default rather than an empty dictionary 

---> fake_extra_scores: Optional[ExtraScores] = None